apps/condo/domains/common/components/Import/Index.tsx

```diff
@@ -156,6 +156,7 @@ const ImportWrapper: React.FC<IImportWrapperProps> = (props) => {
                 const config = getUploadSuccessModalConfig(ImportTitle, message, ImportOKMessage)
                 activeModal.current = modal.success(config)
             }
+            onFinish()
```

Comment:
naming collision, makes confusion, can you please deconstruct `props.onFinish` as `handleFinish`, for example?

Reply:
done